Update Full Text Search because of RelevanceV2 (#1115)
owncloud/docs-server: modules/admin_manual/pages/configuration/server/occ_commands/app_commands/_full_text_search_commands.adoc

[source,plaintext]
----
search

>   search:index:create         Create initial Search Index for one or all users. 

Reviewer: ```suggestion
  search:index:create         Create initial search index for one or all users. 
```

                              This command could not update the search index correctly 
                              after the initial indexing.
  search:index:fillSecondary  Fill a secondary index based on the indexed data we have.
  search:index:rebuild        Rebuild the search index for a given User. 
                              If you want to rebuild the whole index, run

>                               `search:index:reset` and then `search:index:create --all`

Reviewer: ```suggestion
                              `search:index:reset` and then `search:index:create --all`.
```

>   search:index:reset          Reset the index

Reviewer: ```suggestion
  search:index:reset          Reset the index.
```

[source,bash,subs="attributes+"]
----
{occ-command-example-prefix} search:index:fillSecondary [options] [--] <connector_name> <user_id>...
----

=== Arguments

[width="100%",cols="20%,100%"]
|===
| connector_name
| The name of the connector. `Legacy` (default) or `RelevanceV2`.

| `<user_id>` 
| Provide a userId. This argument is required.
|===

=== Options

[width="100%",cols="20%,100%"]
|===
| `-q` `--quiet` 
| Do not output any message.

| `-f` `--force` 
| Use this option to fill the secondary index without further questions.

| `--startOver`
| Start indexing from the beginning, not from a previous savepoint.

| `--chunkSize=CHUNKSIZE`
| The savepoint will be updated after processing this number of files. [default: "100"]
|===

== Rebuild the Full Text Search Index

> Rebuild the indexes for a given User. All the indexes associated with the configured connectors will be rebuilt. This won't apply any change to the configuration of the index if it's already setup but it will setup any index that hasn't been setup yet. Check "search:index:reset" to reset all the indexes associated to the configured connectors

Reviewer: ```suggestion
Rebuild the indexes for a given user. All the indexes associated with the configured connectors will be rebuilt. This won't apply any change to the configuration of the index if it's already set up but it will set up any index that hasn't been set up yet. Check "search:index:reset" to reset all the indexes associated to the configured connectors.
```

> Rebuild the indexes for a given user. All the indexes associated with the configured connectors will be rebuilt. This won't apply any change to the configuration of the index if it's already set up but it will set up any index that hasn't been set up yet. Check "search:index:reset" to reset all the indexes associated to the configured connectors.

Reviewer: ```suggestion
Rebuild the index for a given user. All the indexes associated with the configured connectors will be rebuilt. This won't apply any change to the configuration of the index if it's already set up but it will set up any index that hasn't been set up yet. Check "search:index:reset" to reset all the indexes associated to the configured connectors.
```

== Reset the Full Text Search Index

> Reset all the configured indexes. Changes in the configuration of the indexes associated with the configured connectors will take place. All the data in those indexes will be removed, so you'll need to rebuild the indexes, usually with "search:index:rebuild"

Reviewer: ```suggestion
Reset all the configured indexes. Changes in the configuration of the indexes associated with the configured connectors will take place. All the data in those indexes will be removed, so you'll need to rebuild the indexes, usually with "search:index:rebuild".
```
